packages/notification-center/src/hooks/use-notifications.hook.ts

```diff
@@ -16,10 +16,15 @@ export function useNotifications(props?: IUseNotificationsProps) {
     markAsSeen,
     updateAction: mapUpdateAction,
     refetch: mapRefetch,
+    disposeDefault,
   } = useContext<INotificationsContext>(NotificationsContext);
 
   const notifications = mapNotifications[props?.storeId];
 
+  useEffect(() => {
+    if (props?.storeId) disposeDefault();
+  }, []);
```

Comment:
I think we remove the `disposeDefault` function, and instead of it we at the `notifications-providers.tsx` file not to generate the default store if user has passed stores than it just won't be generated.